app/domain/people_skills_filter.rb

```diff
@@ -28,14 +52,69 @@ def filter_by_rated
   end
 
   def filter_by_level_and_interest(entries)
-    if level.nil? && interest.nil?
-      entries
-    elsif level.nil?
-      entries.where('interest >= ?', interest)
-    elsif interest.nil?
-      entries.where('level >= ?', level)
-    else
-      entries.where('level >= ? AND interest >= ?', level, interest)
+    return entries if levels_and_interests_for_skills.empty?
```

Comment:
what about `return entries unless levels_and_interests_for_skills`

Reply:
same as above, this would not work anymore because unless levels_and_interests_for_skills only checks for 'nil' or 'false'